community/dbms/src/test/java/org/neo4j/commandline/dbms/DumpCommandTest.java

```diff
@@ -128,7 +168,7 @@ public void shouldRespectTheStoreLock() throws IOException, IncorrectUsage, Comm
             execute( "foo.db" );
             fail( "expected exception" );
         }
-        catch ( CommandFailed e )
+        catch ( CommandFailed | IllegalArgumentException e )
```

Comment:
Surely in this test we should know exactly what exception we are catching. Anything else would give a better error (i.e. the stack trace) if it was allowed to percolate up to the top level.